ui-v2/app/templates/components/toggle-button.hbs

```diff
@@ -0,0 +1,2 @@
+<input type="checkbox" id={{concat 'toggle-button-' guid}} onchange={{action 'change'}} />
+<label data-test-toggle-button={{name}} for={{concat 'toggle-button-' guid}}><a href="#" onclick={{action 'click'}}>{{yield}}</a></label>
```

Comment:
Can you do a button instead of an href-less anchor?

Reply:
Oh possibly! We just ignore the click, this is only so you can tab to the link still even though it's a label.  I suppose it's not a hyperlink, so a button would probably make more sense, will take a look 👍

Reply:
What's the need for tabbing to the label? If you just need to put it in the tab order of the document you could add `tabindex="1"` to the `label`, or if you just need to make it programmatically focusable, you could make add `tabindex="-1"`

Reply:
Ah so bit of extra explanation here, this is the Datacenter and Namespace menu trigger, so ideally you need to be able to tab to it (along with all the other things in the main nav menu). I think generally folks shy away from using positive `tabindex`s, it's one of those things that once you start using it rapidly becomes unmanageable. So we want it to remain in the natural tabbing order right now.

Reply:
d'oh I meant "0" for the first one, you're totally right about shying away from positive tab indicies. Is the checkbox in the tab order? I wonder if moving to a button will allow usage of spacebar for toggling instead of enter (looks like enter is what triggers it in chrome).

Reply:
`<button>` along with the spacebar works! Superstar! Thankyou!

Reply:
See https://github.com/hashicorp/consul/pull/6941